src/request.ts

```diff
@@ -452,19 +445,26 @@ class DatastoreRequest {
    *   const entities = data[0];
    * });
    */
-  get(keys, options?): Promise<EntityDataObj[]>;
-  get(keys, options?, callback?): void|Promise<EntityDataObj[]> {
-    if (is.fn(options)) {
-      callback = options;
-      options = {};
-    }
-
-    options = options || {};
+  get(keys: Entities, options?: CreateReadStreamOptions): Promise<Entity|Stream>;
+  get(keys: Entities, callback: GetCallback): void;
+  get(keys: Entities, options: CreateReadStreamOptions, callback: GetCallback): void;
+  get(keys: Entities, optionsOrCallback?: CreateReadStreamOptions|GetCallback, cb?: GetCallback): void|Promise<Entity|Stream> {
+    if (typeof optionsOrCallback === 'function' && typeof optionsOrCallback !== 'object') {
+      cb = optionsOrCallback;
+      optionsOrCallback = {};
+    } else if (typeof optionsOrCallback === 'object' && optionsOrCallback !== null) {}
+    else optionsOrCallback = {};
+
+    let callback: GetCallback;
+    const options: CreateReadStreamOptions = optionsOrCallback;
+
+    if (typeof cb === 'function' && cb !== undefined) callback = cb;
```

Comment:
Always prefer more curly braces and new lines :)

Reply:
Roger that